applications/ParticleMechanicsApplication/custom_strategies/strategies/mpm_residual_based_newton_raphson_strategy.hpp

```diff
@@ -133,17 +133,15 @@ class MPMResidualBasedNewtonRaphsonStrategy
     MPMResidualBasedNewtonRaphsonStrategy(
         ModelPart& rModelPart,
         typename TSchemeType::Pointer pScheme,
-        typename TLinearSolver::Pointer pNewLinearSolver,
```

Comment:
Should the linear solver in line 121 be removed as well in this case?
Or is it only removed in this constructor because the linear solver is already passed to the BuilderAndSolver?
Just for me to understand better.

Reply:
Hi Andino! Yes, exactly, it is removed here because the linear solver is passed in BuilderAndSolver.

Reply:
Got it, thank you!